docs/cheaha/getting_started.md

```diff
@@ -20,14 +20,14 @@ The primary method for accessing Cheaha is through our online portal website, Op
 
 [SSH](../uab_cloud/remote_access.md#command-line-via-ssh) may be used to access Cheaha. Connect to host `cheaha.rc.uab.edu` on port `22`.
 
-### With VSCode
+### With Integrated Development Environments (IDEs)
 
-An alternative method suited for developers using VSCode, is to use the "Remote - Tunnels" extension to connect to an [HPC Desktop Interactive Job](./open_ondemand/hpc_desktop.md). More details on this process are available in the [VSCode Tunnel](./open_ondemand/hpc_desktop.md#visual-studio-code-remote-tunnel) section.
+An alternative method suited for developers is to use IDEs like VSCode, using the "Remote - Tunnels" extension to connect to an [HPC Desktop Interactive Job](./open_ondemand/hpc_desktop.md). You can find more information for setting this up in the [VSCode Tunnel](./open_ondemand/hpc_desktop.md#visual-studio-code-remote-tunnel) section.
```

Comment:
Add a comma after "extension" to make the remote tunnels part a subclause.

Bring "section" into `[VSCode Tunnel]` link but leave "section" lowercase. We use this form somewhat consistently around the docs, so let's stick to it.